usecases/configuration.go

```diff
@@ -88,7 +91,8 @@ func Configure(sys *components.System) ([]json.RawMessage, error) {
 			}
```

Comment:
Above, the printed warning should be done some other way, maybe return an error?
Polluting the console with messages that a user don't know how to resolve provides no value to the user, in the end.

Reply:
And the logic could be rewritten to skip the `err == nil` check. `err == nil` checks are somewhat dangerous, as most Go developers are used to the extremely more common `err != nil`.